int64 start
---
float64 reading1
float64 reading2
float64 reading3
float64 reading4